### PR TITLE
Optimize and complete the document of Colocation with Enhanced QOS

### PR DESCRIPTION
**Pull request by chenkaiyue**
Optimize and complete the document of Colocation with Enhanced QOS.

### CODE REVIEW

File: site/content/en/docs/Getting started/introduction.md [hunk 1]
```diff
@@ -39,8 +39,9 @@ EffectiveHorizontalPodAutoscaler supports prediction-driven autoscaling. With th
 
 Provide a simple but efficient scheduler that schedule pods based on actual node utilization data，and filters out those nodes with high load to balance the cluster. [learn more](/docs/tutorials/scheduling-pods-based-on-actual-node-load).
 
-**Colocation with Enhanced QoS**
+**Colocation with Enhanced QOS**
 
+QOS-related capabilities ensure the running stability of Pods on Kubernetes. It has the ability of interference detection and active avoidance under the condition of multi-dimensional metrics, and supports reasonable operation and custom metrics access; it has the ability to oversell elastic resources enhanced by the prediction algorithm, reuse and limit the idle resources in the cluster; it has the enhanced bypass cpuset Management capabilities, improve resource utilization efficiency while binding cores. [learn more](/docs/tutorials/using-qos-ensurance.md).
```
Comment (qmhu) [inline site/content/en/docs/Getting started/introduction.md:44]: maybe should remove "md", you can test it in local preview

File: site/content/en/docs/Tutorials/QOS/qos-interference-detection-and-active-avoidance.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
+---
+title: "QoS: Interference Detection and Active Avoidance"
```
Comment (qmhu) [inline site/content/en/docs/Tutorials/QOS/qos-interference-detection-and-active-avoidance.md:2]: Please remove all QOS in all title

File: site/content/en/docs/Tutorials/QOS/_index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+---
+title: "QOS"
```
Comment (qmhu) [inline site/content/en/docs/Tutorials/QOS/_index.md:3]: It's not suitable for a directory name

File: site/content/zh/docs/Getting started/introduction.md [hunk 4]
```diff
@@ -41,7 +41,9 @@ EffectiveHorizontalPodAutoscaler 支持了预测驱动的弹性。它基于社
 
 动态调度器根据实际的节点利用率构建了一个简单但高效的模型，并过滤掉那些负载高的节点来平衡集群。[了解更多](/zh-cn/docs/tutorials/scheduling-pods-based-on-actual-node-load)。
 
-**基于 QoS 的混部**
+**基于 QOS 的混部**
+
+QOS相关能力保证了运行在 Kubernetes 上的 Pod 的稳定性。具有多维指标条件下的干扰检测和主动回避能力，支持精确操作和自定义指标接入；具有预测算法增强的弹性资源超卖能力，复用和限制集群内的空闲资源；具备增强的旁路cpuset管理能力，在绑核的同时提升资源利用效率。[了解更多](/zh-cn/docs/tutorials/using-qos-ensurance.zh.md)。
```
Comment (qmhu) [inline site/content/zh/docs/Getting started/introduction.md:46]: remove ".md" and ".zh"

File: site/content/zh/docs/Tutorials/QOS/_index.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+---
+title: "QOS与混布"
```
Comment (qmhu) [inline site/content/zh/docs/Tutorials/QOS/_index.md:3]: 可以想一个更好的标题

File: site/content/zh/docs/Tutorials/QOS/qos-accurately-perform-avoidance-actions.zh.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title: "QoS: 精确执行回避动作"
```
Comment (qmhu) [inline site/content/zh/docs/Tutorials/QOS/qos-accurately-perform-avoidance-actions.zh.md:2]: 所有标题中去掉 Qos

File: site/content/en/docs/Getting started/introduction.md (hunk 1 above)
Comment (chenkaiyue) [reply site/content/en/docs/Getting started/introduction.md:44]: ok

File: site/content/en/docs/Tutorials/QOS/qos-interference-detection-and-active-avoidance.md (hunk 2 above)
Comment (chenkaiyue) [reply site/content/en/docs/Tutorials/QOS/qos-interference-detection-and-active-avoidance.md]: done

**Review by qmhu — APPROVED** [review]

**Pull request merged**